$OpenBSD: patch-src_main_c,v 1.1 2012/10/12 09:09:46 kirby Exp $

Properly call gdk_threads_enter() and gdk_threads_leave() to prevent
abort() on startup due to gtk trying to unlock a uninitialized
mutex

--- src/main.c.orig	Thu Oct 11 19:40:20 2012
+++ src/main.c	Thu Oct 11 19:40:41 2012
@@ -124,7 +124,9 @@ main (int argc, char *argv[])
 	
 	init();
 	
+	gdk_threads_enter();
 	gtk_main ();
+	gdk_threads_leave();
 
 
 	
